universum/modules/launcher.py

```diff
@@ -455,3 +456,25 @@ def _is_conditional_step_with_children_present(
                Launcher._is_conditional_step_with_children_present(step.if_failed):
                 return True
         return False
+
+    def _warn_about_critical_conditional_steps(
+            self, configuration: Optional[configuration_support.Configuration]) -> None:
+        step_names: List[str] = []
+        Launcher._append_critical_conditional_step_names_recursively(configuration, step_names)
+        if step_names:
+            step_names_str: str = ",".join(step_names)
+            self.out.log(f"WARNING: 'critical' flag will be ignored for conditional steps: [{step_names_str}]. "
```

Comment:
Could you make a list where every step name is on new line? I presume it would be quite unreadable the way it is printed now.